### ISSUE TITLE
Order of the controls

### ISSUE THREAD

**Issue by karasma3** [chris-m92/react-leaflet-custom-control#2]
Hello there. 
I am trying to migrate from react-leaflet v2 to v3 and I was using [https://github.com/LiveBy/react-leaflet-control](https://github.com/LiveBy/react-leaflet-control) package. This worked perfectly, but it is not compatible with v3.

So I switched to your package instead and tried to make it work. It works same as the other package, but the thing is the order of the controls is not **preserved**!

Before: react-leaflet 2.8.0, react-leaflet-control 2.1.2
``` react
import Control from 'react-leaflet-control'
```
Now: react-leaflet 3.2.5, react-leaflet-custom-control 1.2.2
``` react
import Control from 'react-leaflet-custom-control'
```
The code is basically unchanged, now just using MapContainer instead of Map component.
``` react
<MapContainer>
  <ScaleControl position="bottomright" />
  <ZoomControl position="bottomright" />
  <Control position="bottomright">
    <MyComponentGps />
  </Control>
  <Control position="bottomright">
    <MyComponentHide />
  </Control>
  <MeasureControl position="bottomright" />
  <PolylineMeasure position="bottomright" />
</MapContainer>
```
With react-leaflet 2.8.0 and react-leaflet-control 2.1.2 it worked as intended: the order was from bottom to top - Scale, Zoom, MyComponentGps, MyComponentHide, MeasureControl, PolylineMeasure.
![image](https://user-images.githubusercontent.com/17758032/160621518-efa35b10-9e74-4bff-902d-4dad0e3a7ab8.png)

But with react-leaflet 3.2.5 and react-leaflet-custom-control 1.2.2 the order is different... From bottom to top - MyComponentHide, MyComponentGps and after that there are Scale, Zoom, MeasureControl, PolylineMeasure.

![image](https://user-images.githubusercontent.com/17758032/160622131-5c4ff1a3-649e-4f00-b99f-9e428ccabf8b.png)

Can you help me figure it out how to use it right, or how to tweak your code so that leaflet controls would be added to map in the order?

**Comment by chris-m92** [comment]
I agree. The way I got it to work was through React's `createPortal` which definitely doesn't play nicely with Leaflet. I've found that manually adding in the leaflet controls in order with your custom controls helps to preserve the order that you're looking for. 

So when you initialize your map, set attribution and zoom control to false and re-add in the order that you want them top to bottom. If you have any better ideas that would preserve the original v3 ordering similar to what v2 was able to do, I'd love to know because it's an annoying issue that I'm not sure how to hack it.

```js
<MapContainer
  attributionControl={false}
  zoomControl={false}
>
  <PolylineMeasure />
  <MeasureControl />
  <MyComponentHide />
  <MyComponentGps />
  <ZoomControl />
  <AttributionControl />
</MapContainer>
```

**Comment by chris-m92** [comment]
## Released in v1.2.5

Based on this [Stack Overflow Post](https://stackoverflow.com/questions/52658703/make-reactdom-createportal-insert-element-as-first-child-of-target-container-in), the control now appends or prepends a container `div` element to the desired control position container based on the new `prepend` prop. That is then used as the portal target. Additional information is added to the `README` to explain this.
